README.md

```diff
@@ -13,7 +13,7 @@ semantic API, and a lower level HTTP API for extending behaviour.
 
 #### Cargo.toml
 ```toml
-octocrab = "0.16"
+octocrab = "0.17"
```

Comment:
I think I'd rather point to the new cargo command which is evergreen, and keep the links below to latest.

```suggestion
cargo add octocrab
```

Reply:
Or maybe just including this badge which will always show the current published cargo version?

![crates.io](https://img.shields.io/crates/v/octocrab.svg)